python/nmodl/ode.py

```diff
@@ -26,10 +26,27 @@
 if not ((major >= 1) and (minor >= 2)):
     raise ImportError(f"Requires SympPy version >= 1.2, found {major}.{minor}")
 
-# Some functions are protected inside sympy, if user has declared such a function, it will fail
-# because sympy will try to use its own internal one.
-# Rename it before and after to a single name
+# Some identifiers are protected inside sympy, if user has declared such a function, it will fail
+# because sympy will try to use its own internal one; or error out for invalid variables.
+# Rename it before and after to a unique name.
 forbidden_var = [
+    # Python keywords
```

Comment:
You can use https://docs.python.org/3/library/keyword.html#keyword.kwlist instead of hardcoding these.

Reply:
i really don't want to.